File: strategies/kotlin/StrategyBinanceOrder.kt
import com.binance.api.client.BinanceApiClientFactory

fun main() {
    val factory = BinanceApiClientFactory.newInstance("KEY", "SECRET")
    val client = factory.newRestClient()
    client.newOrderTest(NewOrder.limitBuy("BTCUSDT","0.01","29000"))  // test endpoint
}

File: strategies/haskell/StrategyBinanceOrder.hs
-- Strategy: Binance Order
-- Language: Haskell
-- Exchange: Binance (Spot)

import Binance.Client

main :: IO ()
main = do
  client <- newClient "KEY" "SECRET"
  resp <- placeOrder client LimitBuy "BTCUSDT" 0.01 (Just 29000)
  print resp
